server/lib/csp.js

```diff
@@ -2,30 +2,41 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-// middleware to take care of CSP. CSP headers are only sent in development
-// mode, and are not sent if the user is running the unit tests.
+// Middleware to take care of CSP. CSP headers are not sent unless
+// config option 'cspEnabled' is set, with a special exception for
+// the /tests/index.html path, which are the frontend unit tests.
```

Comment:
Not sure if we want to add the comment about CSP headers being _always_ sent in development, regardless of what it's set to in the local.json file.